ui/src/main/js/page/stocks/index.js

```diff
@@ -0,0 +1,180 @@
+'use strict';
+
+import React from 'react';
+import StockList from '../../component/stock-list/index.js';
+
+const ReactDOM = require('react-dom');
+const client = require('../../api/client');
+const follow = require('../../api/follow');
+const stompClient = require('../../websocket-listener');
+const when = require('when');
+const root = 'https://localhost/stock-service/api';
+
+//allow react dev tools work
+window.React = React;
+
+class Stocks extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {stocks: [], attributes: [], pageSize: 30, links: {}};
+    this.updatePageSize = this.updatePageSize.bind(this);
+    this.onNavigate = this.onNavigate.bind(this);
+    this.refreshCurrentPage = this.refreshCurrentPage.bind(this);
+    this.onSearch = this.onSearch.bind(this);
+  }
+  componentDidMount() {
+    this.loadFromServer(this.state.pageSize);
+    stompClient.register([
+      {route: '/topic/updateStock', callback: this.refreshCurrentPage}
+    ]);
+  }
+	loadFromServer(pageSize) {
+		follow(client, root, [
+				{rel: 'stocks', params: {size: pageSize, sort:'percentageChange,desc'}}]
+		).then(stockCollection => {
+				return client({
+					method: 'GET',
+					path: stockCollection.entity._links.profile.href,
+					headers: {'Accept': 'application/schema+json'}
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/modernmaster/forecaster/pullRequest?prid=7238741)
```suggestion
					headers: {"Accept": 'application/schema+json'}
```